### ISSUE TITLE
Details about results in the paper

### ISSUE THREAD

**Issue by Rhyssiyan** [renmengye/inc-few-shot-attractor-public#1]
Hi, thanks for your good job. It's a very interesting work.
I want to know more details about Table 2 in your paper. What does the Acc in the table mean?  Top-1 acc or Top-5 acc? 
In addition, I'm also very curious about the Train-A-Val top-1 accuracy  of your resnet-18 model pretrained on the Train-A-Train part.

Thanks in advance.

**Comment by renmengye** [comment]
The Acc means top-1 accuracy. Train-A-Val top-1 is around 62.4%.
